packages/bpk-component-infinite-scroll/src/withInfiniteScroll.js

```diff
@@ -182,7 +213,6 @@ const withInfiniteScroll = (
               index: nextIndex,
               elementsToRender: (elementsToRender || []).concat(nextElements),
               showSeeMore: seeMoreAfter === index / elementsPerScroll,
-              isListFinished: false,
```

Comment:
I don't really understand why this was removed, could you explain it?

Reply:
Because this is not needed: it's already initialised to `false`, and it's only changed to `true` when the results finish or when the request fails, so there's no need to set it to `false` again every time more results are fetch.

Not a big deal, but makes the code slightly cleaner :-P

Reply:
Sounds good to me, thanks for explaining.

Reply:
I don't think this can be removed. If the datasource changes and the previous list has been finished `isListFinished` would have been set to false.